tests/compare_hilites.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+model_formatter="/home/natasha/MurfelClickHouse/build/programs/clickhouse-format";
+new_formatter="/home/natasha/copyClickHouse/build/programs/clickhouse-format";
+compare="/home/natasha/copyClickHouse/tests/a.out"
```

Comment:
We usually use environmental variables or command line arguments to pass paths to a script which cannot be determined by that script. I mean `model_formatter`, `new_formatter`, `compare`, `flags`.